h/static/scripts/annotation-sync.coffee

```diff
@@ -53,11 +53,6 @@ module.exports = class AnnotationSync
       this._syncCache(channel)
     @bridge.onConnect(onConnect)
 
-  # Provide a public interface to the annotation cache so that other
-  # sync services can lookup annotations by tag.
-  getAnnotationForTag: (tag) ->
-    @cache[tag] or null
-
```

Comment:
This was only used by `annotation-ui-sync` and this pr removes the part of `annotation-ui-sync` that was using it, so this is simply deleted